components/images.css

```diff
@@ -0,0 +1,22 @@
+.img {
+  max-width: 50rem;
```

Comment:
you can add object-fit : contain /cover to make it more precise and responsive everything  else look fine

Reply:
Thank you. done with object-fit property